Makefile

```diff
@@ -27,6 +50,9 @@ clean:
 vm-protobuf:
 	cd $(CURDIR)/vm/protobuf; protoc --proto_path=. --go_opt=paths=source_relative --go_out=plugins=grpc:. vm.proto
 
+insight-protobuf:
```

Comment:
I don't think this is needed anymore since we are using protobuf from auto-policy-discovery